packages/components/workbench/src/WorkbenchHeader/WorkbenchHeader.styles.ts

```diff
@@ -0,0 +1,40 @@
+import { css } from 'emotion';
+import tokens from '@contentful/f36-tokens';
+
+export const getWorkbenchHeaderStyles = (hasBackButton = false) => ({
+  workbenchHeader: css({
+    display: 'flex',
+    alignItems: 'center',
+    flexShrink: 0,
+    width: '100%',
+    height: '70px',
+    padding: `0 ${tokens.spacingL}`,
+    borderBottom: `1px solid ${tokens.gray300}`,
+    backgroundColor: tokens.gray100,
+    zIndex: tokens.zIndexWorkbenchHeader,
+    boxSizing: 'border-box',
+    ...(hasBackButton && { paddingLeft: 0 }),
```

Comment:
IMO it would look better if you pass an array to the `css()` method here, instead of manually destructuring

Reply:
do you mean?

```ts
css([{ ... }, hasBackButton && { paddingLeft: 0 }])
```

I usually prefer to do that in the components, but I thought the team would prefer me doing all this emotion stuff in the getStyles function

Reply:
yeah, I just think that approach looks better than the destructuring one, but isn't a big issue anyway 😄 
Would also be fine with having a `hasBackButton` style and merging it with `cx` on the component as you said